tensorboard/uploader/uploader_main.py

```diff
@@ -428,6 +538,24 @@ def execute(self, server_info, channel):
         sys.stderr.flush()
 
 
+def _raise_if_bad_experiment_name(name):
+    if name and len(name) > _EXPERIMENT_NAME_MAX_CHARS:
+        raise ValueError(
```

Comment:
Can we `_die` instead of raising `ValueError` here, to avoid showing the
user a stack trace? (Maybe rename `_raise_if_*` to `_die_if_*`, too.)